src/Kunstmaan/MediaBundle/Resources/config/serializer/Entity.Media.yml

```diff
@@ -0,0 +1,23 @@
+Kunstmaan\MediaBundle\Entity\Media:
+    exclusion_policy: ALL
+    properties:
+        id:
+            expose: true
+        name:
+            expose: true
+        description:
+            expose: true
+        copyright:
+            expose: true
+        url:
+            expose: true
+        originalFilename:
```

Comment:
AFAIR the JMS serializer doesn’t do camel case to snake conversion, so the field name needs to be the same as on the entity.